packages/backend/server/src/core/sync/gateway.ts

```diff
@@ -280,12 +280,15 @@ export class SpaceSyncGateway
     const adapter = this.selectAdapter(client, spaceType);
 
     // TODO(@forehalo): we might need to check write permission before push updates
-    const timestamp = await adapter.push(
-      spaceId,
-      docId,
-      updates.map(update => Buffer.from(update, 'base64')),
-      user.id
-    );
+    let timestamp = 0;
+    for (const update of updates) {
```

Comment:
Why was batch processing changed to serial? Was there a change in the underlying implementation?

Reply:
the updates will always be merged before sending to backend, and batch push api was deprecated